Cargo.toml

```diff
@@ -5,15 +5,16 @@ authors = ["Adrian Benavides <adri.benavides312@gmail.com>"]
 repository = "https://github.com/adrianbenavides/bb8-lapin"
```

Comment:
You should bump the version to `0.3.0` and add a new entry to the `changelod.md` file.

Reply:
Great, I added an entry for 0.2.0 to the changelog as well for completeness.